<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>364</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dp93-calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/icon.svg</normaloff>:/icon/icon/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="body">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #000000;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="content">
      <item>
       <layout class="QHBoxLayout" name="sectionDisplay">
        <item>
         <widget class="QLineEdit" name="calcDisplay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #000000;
    color: #ffffff;
    font-size: 50px;
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>9</number>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonsOne">
        <item>
         <widget class="QPushButton" name="btnAC">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #888888;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #B3B3B3;
}

QPushButton:pressed {
    background-color: #666666;
}</string>
          </property>
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNeg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #888888;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #B3B3B3;
}

QPushButton:pressed {
    background-color: #666666;
}</string>
          </property>
          <property name="text">
           <string>+/-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPerc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #888888;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #B3B3B3;
}

QPushButton:pressed {
    background-color: #666666;
}</string>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #FFC200;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #FFD85A;
}

QPushButton:pressed {
    background-color: #DBA700;
}</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonTwo">
        <item>
         <widget class="QPushButton" name="btnNum7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMulti">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #FFC200;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #FFD85A;
}

QPushButton:pressed {
    background-color: #DBA700;
}</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonThree">
        <item>
         <widget class="QPushButton" name="btnNum4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMinus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #FFC200;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #FFD85A;
}

QPushButton:pressed {
    background-color: #DBA700;
}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonFour">
        <item>
         <widget class="QPushButton" name="btnNum1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNum3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #FFC200;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #FFD85A;
}

QPushButton:pressed {
    background-color: #DBA700;
}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonFive">
        <item>
         <layout class="QHBoxLayout" name="blockButtonOne">
          <item>
           <widget class="QPushButton" name="btnNum0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 32px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="blockButtonTwo">
          <item>
           <widget class="QPushButton" name="btnFloat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #555555;
    color: #ffffff;
    border: none;
    border-radius: 32px;
    font: 30px;
}

QPushButton:hover {
    background-color: #737373;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
            </property>
            <property name="text">
             <string>,</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEqual">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #FFC200;
    color: #ffffff;
    border: none;
    border-radius: 33px;
    font: 30px;
}

QPushButton:hover {
    background-color: #FFD85A;
}

QPushButton:pressed {
    background-color: #DBA700;
}</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
